ui/src/app/pages/workspace/workspace-edit.tsx

```diff
@@ -2396,10 +2505,22 @@ export const WorkspaceEdit = fp.flow(
                         {errors.otherPopulationDetails && (
                           <li>{errors.otherPopulationDetails}</li>
                         )}
+                        {errors.aianResearchType && (
+                          <li>
+                            You must select the type of AIAN research you are
+                            conducting (Question 6.1)
+                          </li>
+                        )}
+                        {errors.aianResearchDetails && (
+                          <li>
+                            You must provide details about your study design
```

Comment:
needs to distinguish between the two ways to get this error: missing and too-long-text

Reply:
Thanks for catching this